### ISSUE TITLE
rpi4 mit S.USV bootet nicht mit Rapberrymatic 3.73.9.20231130

### ISSUE THREAD

**Issue by falohome** [jens-maus/RaspberryMatic#2553]
### Describe the issue you are experiencing

Mit der Raspberrymatic Version 3.73.9.20231130 bootet der rpi4 mit installierter S.USV nicht mehr.
Wird die S.USV entfernt, startet der Bootvorgang. 

### Describe the behavior you expected

Rasberrymatic bootet mit S.USV.

### Steps to reproduce the issue

1. SD-Karte mit RPM 3.73.9.20231130 flashen.
2. S.USV montiert.

### What is the version this bug report is based on?

3.73.9.20231130

### Which base platform are you running?

rpi4 (RaspberryPi4)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Kein Logfile vorhanden.
```

### Additional information

Ein angeschlossener Monitor zeigt keinerlei Ausgaben. Alle vorherigen Versionen von Raspberrymatic funktionierten bisher ohne Fehler.
Bootloader (eeprom) des rpi4 ist aktuell.

**Comment by Linustrispoke** [comment]
Hallo zusammen, auf einem Raspi 3B verhält es sich genauso. Bootvorgang wird offenbar nicht richtig angestoßen bzw. bleibt in einer Enlosschleife gleich am Anfang hängen. Dadurch wird die Netzwerkkarte/ das Netzwerk erst garnicht aktiv und deshalb kein Zugriff über ssh. Das S.USV-Board bleibt bei dem Versuch der Firmwareerkennung "hängen" (PSU GREEN (Blinkend) Startup – Initialisierung der S.USV Firmware).
Deshalb wieder zurück auf die Oktoberversion, hier läuft alles ohne Probleme.

**Comment by Balming** [comment]
Hallo, ich kann mich Linustrispoke nur anschließen. Gleiches Setup wie oben, Raspi 3B mit S.USV Stromversorgung. Kein Booten. Erst wenn die S.USV ausgebaut ist läuft die neue Firmware. Nach Installieren der alten Firmware wieder alles ohne Probleme

**Comment by DeusUli** [comment]
Habe das gleiche Problem mit meiner Raspimatic. Hatte gehofft die aktuelle 3.73.9 Release hätte das Problem nicht. Leider immer noch so. Also wieder zurück zur Oktober Release.

**Comment by Linustrispoke** [comment]
Hallo noch mal, habe es mit der aktuellen Version noch einmal versucht, mit negativem Erfolg. Habe mir dabei die SD-Karte zerschossen ist aber eine andere Geschichte. Nur mit der November 2023 Version wird die S.USV reichtig erkannt bzw. gestartet.

**Comment by Baxxy13** [comment]
Aktuell hat sich wohl noch niemand mit dem Problem befasst.
Bei mir liegt es daran das ich keine S.USV habe und da somit auch nichts testen kann.
Ob @jens-maus eine hat weiß ich nicht.

Für den Anfang wäre es ganz hilfreich wenn jemand mal sein System mit einer Tastatur bestückt und am Monitor bootet.
Idealerweise dann abfotografieren was zu sehen ist damit man mal einen Anhaltspunkt bekommt.

**Comment by jens-maus** [comment]
Hab das Problem gerade hier mit meiner Test S.USV nachvollziehen können. Konnte den Fehler auch bereits eingrenzen. Es scheint wohl am U-Boot bootloader zu liegen. Sobald man die `u-boot.bin` einer alten Version < 3.73.9.20231130 nutzt und ins `/boot` kopiert startet der Pi wieder korrekt. Muss ich mir also mal anschauen was da genau bei den neueren U-boot versionen/konfigs aktuell den Start verhindert wenn das S.USV auf den GPIO Bus gesteckt ist.

**Comment by Linustrispoke** [comment]
@Balming @Baxxy13 @DeusUli Hallo zusammen, erst mal schöne Osterfeiertage. Hat schon jemand die März-Version ausprobiert?, mit welchem Erfolg?

**Comment by Baxxy13** [comment]
Der Status ist auch mit der aktuellen Version unverändert. Ansonsten wäre das Ticket hier auch geschlossen.

**Comment by Linustrispoke** [comment]
@jens-maus Hallo nach langer Zeit ....
Hat sich bzgl. dem Problem etwas getan?, würde mein System gerne aktualisieren.

**Comment by jens-maus** [comment]
Nein, da hat sich nix getan und wird sich vmtl auch nix mehr tun. Der ehemalige Hersteller des S.USV hat das Ding eh aufgegeben und da wird nix mehr kommen oder gar produziert. Nimm das Ding runter, schmeiß es weg und besorg die am besten ne vollwertige, externe USV und schließe den Pi daran an. Alles andere lohnt sich vmtl nicht mehr...
